java/PaimonCDCSink/README.md

```diff
@@ -0,0 +1,165 @@
+
+This example parses runtime parameters according to the following rules and passes the parsed parameters to Apache Paimon Actions.
+
+- The Paimon CDC ingestion action name is parsed from the key named action in the 'ActionConf' parameter group.
+- Some global or common parameters can be placed in the 'ActionConf' parameter group. The parameter names should refer to the specific ingestion [action name](https://paimon.apache.org/docs/1.0/cdc-ingestion/overview/).
+- For parameters like 'table_conf' and 'catalog_conf' that are set in the format of Key=Value, the name of the parameter group can be customized, such as “TableConf” or “CatalogConf”. 
+For specific parameter names within the parameter group, they should follow the format “parameter group name@_parameter Key”, 
+such as “table_conf@_bucket”, and the parameter value should be the corresponding Value.
+
+
+Runtime parameters(Sample):
+
+| Group ID      | Key                                        | Description                                                                            | 
+|---------------|--------------------------------------------|----------------------------------------------------------------------------------------|
+| `ActionConf`  | `action`                                   | Name of Apache Paimon CDC ingestion, `kafka_sync_database`, `mysql_sync_database` etc. |
+| `ActionConf`  | `database`                                 | Target Paimon database name.                                                           |
+| `ActionConf`  | `primary_keys`                             | (Optional) The primary keys for Paimon table                                           |
+| `KafkaConf`   | `kafka_conf@_properties.bootstrap.servers` | Bootstrap servers of the Kafka Cluster.                                                |
+| `KafkaConf`   | `kafka_conf@_properties.auto.offset.reset` | Offset of the Kafka Consumer                                                           |
+| `KafkaConf`   | `kafka_conf@_properties.group.id`          | Consumer group Id                                                                      |
+| `CatalogConf` | `catalog_conf@_metastore.client.class`     | Paimon Hive Catalog metastore client class name                                        |
+| `CatalogConf` | `...`                                      | ...                                                                                    |
+| `TableConf`   | `table_conf@_bucket`                       | Bucket of Paimon table                                                                 |
+| `TableConf`   | `...`                                      | ...                                                                                    |
+
+All parameters are case-sensitive.
+
+### Samples
+**Create an MSF application**
```

Comment:
This is not consistent with what we are doing with all other examples.
The goal of these examples is not to cover step by step the creation of an application. There is nothing specific to this example here, except passing the profile name.

Reply:
These instructions do not really explain what data the kafka source is expecting. What schema? What serialization?